reference/functions/findlocalusers.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+title: findlocalusers
+published: true
+---
+
+[%CFEngine_function_prototype(filter)%]
+
+**Description:** Returns a data container of all local users with their attributes that are matching a filter. If no filter is specified, it will return all the local users.
+
+[%CFEngine_function_attributes(filter)%]
+
+The `filter` argument can be used to look up users with specific attributes that match values. The filter is a `"data"` container or `"slist"` comprised of pairs of attribute and value/regex pattern `{ "attribute1=value1", "attribute2=value2", ... }`.
+
+The possible attributes are:
+* `name`: name 
+* `uid`: user id     
```

Comment:
These 2 lines have trailing whitespace, that's why the markdowner GH Action check is failing. Remove the spaces at the end of the line and amend your commit.